build/vm-monitor/Dockerfile

```diff
@@ -8,14 +8,14 @@ RUN apk add musl-dev git
 #
 # Note: unlike in the stackoverflow post we access the branches endpoint so that
 # `latest_commit` will change upon a commit in any branch
-ADD "https://api.github.com/repos/neondatabase/vm-monitor/branches" latest_commit
+ADD "https://api.github.com/repos/neondatabase/neon/branches" latest_commit
 
-RUN git clone https://github.com/neondatabase/vm-monitor.git 
-RUN cargo build --release --manifest-path vm-monitor/Cargo.toml
+RUN git clone https://github.com/neondatabase/neon.git
```

Comment:
A couple things:

1. Should be `--depth 1` so we're not pulling in all of the history as well
2. Branch should be configurable (e.g. via build arg), to make it easier to test vm-monitor changes with autoscaling changes.